### PR TITLE
Feature/dynamic circle color

### PR DESCRIPTION
**Pull request by tranthanhhoa**

### CODE REVIEW

**Comment by toubou91** [comment]
Hi,

thank you for this one, but it seems that the following part (preferred individually, rather than using the respective <code>data-</code> attribute) doesn't work:

```
$("[id$='circle']").percircle({
	progressBarColor: "green"
});
```

Could you please investigate it?

File: src/js/percircle.js [hunk 1]
```diff
@@ -25,12 +25,23 @@
         
         var rotationMultiplier = 3.6;
 
-
         // for each element matching selector
         return this.each(function(){
             var percircle = $(this);
             var progressBarColor = '';
 
+            // When user try to custom progress bar color, should be update color text as well.
```
Comment (tranthanhhoa) [inline src/js/percircle.js]: My grammar is bad, fix it pls :)

**Comment by tranthanhhoa** [comment]
@toubou91 Updated :)

**Comment by toubou91** [comment]
Hey @tranthanhhoa ,

I decided to merge as soon as possible, as I noticed that @ahmadajmi prepares something [cool](https://github.com/ahmadajmi/percircle/commit/9d8d4f3ee53b20b26e27944c67fbdfc72aeb92da) (and as I am not sure about my availability in the upcoming days, it would be better to eliminate the chances for bigger conflicts).

Thus, there are still some [minor issues left](https://github.com/toubou91/percircle/labels/minor), for the case you still want to continue contributing.

Thank you!

**Pull request merged**